Crashlytics/Crashlytics/FIRCrashlytics.m

```diff
@@ -157,6 +169,13 @@ - (instancetype)initWithApp:(FIRApp *)app
       [sessions registerWithSubscriber:self];
     }
 
+    if (remoteConfig) {
+      FIRCLSDebugLog(@"Registering RemoteConfig SDK subscription for rollouts data");
+
+      _remoteConfigManager = [[FIRCLSRemoteConfigManager alloc] initWithRemoteConfig:remoteConfig];
+      [remoteConfig registerRolloutsStateSubscriber:self for:@"firebase"];
```

Comment:
The @"firebase" namespace. Should this be Crashlytics?

Or if "firebase" is like the main app's namespace, can we define it in the Interop SDK as a constant that we import and use here?

Reply:
Yeah I think @danasilver will eventually make the namespace public in introp, right now the namespace is under RC SDK. As Dana suggested, I can make a TODO note after the refactor done.

Reply:
It's the latter (main app's namespace). There's currently a definition for it in the RC SDK that we can move to the Interop or copy over https://github.com/firebase/firebase-ios-sdk/blob/c04601149c7f55ccdda181cfd88a9f847e8a4b30/FirebaseRemoteConfig/Sources/Public/FirebaseRemoteConfig/FIRRemoteConfig.h#L24

Reply:
Awesome - sounds good!